src/Bravo/Serialization.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+#include <nlohmann/json.hpp>
+
+#include "../Data.h"
+#include "../BinaryCoding.h"
+
+#include <vector>
+#include <set>
+#include <string>
+
+namespace TW::Bravo {
+    class Serializable
+    {
```

Comment:
Don't indent contents of namespace. Also keep the opening brace on the same line. Run clang-format when in doubt.

Reply:
Okay